rest/data/project_settings.go

```diff
@@ -438,6 +441,23 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 			}
 		}
 	}
+
+	// If we're just enabled the project, we should run the repotracker to kick things off.
```

Comment:
We do already have:
> If a Project Administrator wants Evergreen to discontinue or start
tracking a project, it can be changed via the Enabled and Disabled radio
buttons.

which I think suffices since this fits that correct behavior